pyproject.toml

```diff
@@ -50,3 +51,59 @@ include = [
 exclude = [
   "tests/*",
 ]
+
+[tool.hatch.envs.default]
+dependencies = [
```

Comment:
Does dependabot work on those?

Reply:
No

Reply:
this project uses dependabot, so I guess it should replace by https://github.com/renovatebot/renovate ?

Reply:
seems stuck in the past too https://docs.renovatebot.com/configuration-options/#python